install

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
This entire file I do not think is appropriate for the tsds-services package. Part of what it does is configure some bits of tsds, but it also does things related to simp, grafana, and vce. I think this belongs in the VCE package, not here.

Reply:
This file is just a list of commands that required to be run one after the other. It is just a time-saving script. It will be deleted in the next commit.